wear/src/main/java/io/homeassistant/companion/android/conversation/ConversationViewModel.kt

```diff
@@ -0,0 +1,40 @@
+package io.homeassistant.companion.android.conversation
+
+import android.app.Application
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.viewModelScope
+import dagger.hilt.android.lifecycle.HiltViewModel
+import io.homeassistant.companion.android.common.data.integration.IntegrationRepository
+import io.homeassistant.companion.android.common.data.websocket.WebSocketRepository
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class ConversationViewModel @Inject constructor(
+    application: Application,
+    private val integrationUseCase: IntegrationRepository,
+    private val webSocketRepository: WebSocketRepository
+) : AndroidViewModel(application) {
+
+    var speechResult = mutableStateOf("")

```

Comment:
By using `by mutableStateOf("")` instead of `= mutableStateOf("")` you can stop using `.value` to access/update these variables.

(This will break updating the speech result from the activity, which shouldn't have been possible in the first place and should use a function in the viewmodel.)

Reply:
ok done, updated all variables and now updating the results in a function instead of the variable directly

Reply:
Now that you're using a function why remove `private set` for `speechResult`?

Reply:
Somewhere along the way android studio asked me to make it public, I will add it back.